ci-operator/step-registry/telcov10n/functional/hcp-cnf/telcov10n-functional-hcp-cnf-workflow.yaml

```diff
@@ -4,6 +4,8 @@ workflow:
     pre:
       - ref: telco5g-jobs-router
       - ref: telco5g-hcp-cluster-setup
+    test:
```

Comment:
As a best practice, it's better to have the test stanza in the config itself instead of in the workflow, that will make the workflow reusable with different tests if needed. 
let me know if this is not relevant to you.

Reply:
Thanks for the comment, I believe for now this is not relevant.